### PR TITLE
[IMP] barcode: update hardware setup doc

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is an [IMP] to add setup sections for Honeywell and Cipherlab scanners in the `hardware` doc in Odoo's Barcode documentation, specific to 17.0.

Here's what I did:
- Added input from both previous doc task (and linked customer support tickets) to clarify setup process for Honeywell and Cipherlab scanners in Odoo Barcode
- Added screenshots from Honeywell and Cipherlab user manuals to display scanner screens and options

### CODE REVIEW

**Comment by brse-odoo** [comment]
Hi @odoo/barcode-doc-review, this PR is ready for content peer review, when one of you has a chance. Please note that the lines that need to be reviewed are lines 109-159 (the two new sections I added).

The rest of the doc was previously-approved and merged yesterday, in [PR #7580](https://github.com/odoo/documentation/pull/7580). Thank you!

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/barcode/setup/hardware.rst:113]: I asked @mejg-odoo for extra context on why we need these configurations to prevent errors. Feel free to modify my suggestion as you see fit!
```suggestion
When using Honeywell scanners, change the *wedge method* settings to *Keyboard* so that the device
can process barcodes as keyboard characters and symbols.
.```

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/barcode/setup/hardware.rst:131]: Totally optional change for brevity
```suggestion
After completing the steps, conduct a test scan to verify the Honeywell scanner is working as intended.
```

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/barcode/setup/hardware.rst:136]: I asked barcode expert @dara-odoo for his insight, and he mentioned:
- prefers to not say "prevent errors" because it implies that Odoo might have errors.
- compared to my above suggestion about keyboard characters and symbols, he doesn't think explaining it in such detail is necessary. He'd personally keep it simple and say "it just works"

Feel free to consider these suggested rewrites (more detail or keeping it simple) and decide which angle you'd like to go with in your PR, Brandon (:

```suggestion
When using Cipherlab scanners, follow the instructions below to ensure barcodes can be scanned in
Odoo.
```

**Review by Felicious — APPROVED** [review]
Hey @brse-odoo,

Great job on testing and updating docs about hardware that's difficult to test :smile: 

I just had some minor suggestions about adding more context as to why we're performing certain instructions, but they're totally optional.

After you take a look at the comments, feel free to pass it along to the next stage of review!

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/barcode/setup/hardware.rst:113]: That logic makes sense to me! Suggestion implemented ✅

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/barcode/setup/hardware.rst:131]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/barcode/setup/hardware.rst:136]: I understand the logic about changing the language in the opening sentence for each section, and I actually agree with "keeping it simple" and then letting the reader follow the instructions below. So, I'm going with that suggestion 👍

Suggestion implemented ✅

**Comment by brse-odoo** [comment]
Thank you @Felicious for the quick review!

Alright @odoo/us-doc-review, this PR has passed the peer review round, and is now ready for the final content review round. Please feel free to reach out to me if you have any questions. Thank you!

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/barcode/setup/hardware.rst:129]:
```suggestion
Locate the :guilabel:`Wedge Method` setting. By default, this is set to :guilabel:`Standard`.

.. image:: hardware/hardware-honeywell-settings.png
   :align: center
   :alt: Data processing settings options for Honeywell scanner.

Change the :guilabel:`Wedge Method` setting to :guilabel:`Keyboard`.
```
OPTIONAL: Since the screenshot you used *still* shows the 'Standard' setting -- it feels like the placement of the image should come *before* mentioning that it needs to be switched to 'Keyboard.'

With this ordering, the reader will be able to follow along just a little bit easier. At least, that's how I *hope* they'd feel! haha what are your thoughts?

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/barcode/setup/hardware.rst:142]:
```suggestion
:guilabel:`⚙️ (gear)` icon over a blue barcode icon.
```

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/barcode/setup/hardware.rst:153]:
```suggestion
By default, the :guilabel:`Input Method`, under :guilabel:`Keyboard Emulation`, is set to
```

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/barcode/setup/hardware.rst:161]: Do you think there's an opportunity for a :seealso: block here at the bottom to related Barcode docs? Could be a nice way to round out the doc...what do you think?

**Review by ksc-odoo — APPROVED** [review]
Great work on this PR, @brse-odoo -- Just finished my Final Review, and am approving it now. There are only a few things that require your attention, but I don't anticipate you having any issues. Once it's ready, feel free to tag this for Tech Review. Once again, fantastic job on this doc. 👍

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/barcode/setup/hardware.rst:129]: Yeah, I totally get that logic. Because I had to rely on user manuals for some of these my options were pretty limited 😅 But I agree that changing the placement could help avoid confusion, so I'll change it.

Suggestion implemented ✅

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/barcode/setup/hardware.rst:142]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/barcode/setup/hardware.rst:153]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/barcode/setup/hardware.rst:161]: Yeah, that's a good idea. I'll link the one other doc in this subsection (`software`) to round it out.

Suggestion implemented ✅

**Comment by brse-odoo** [comment]
Thank you @ksc-odoo for the review! Alright @samueljlieber, this PR has passed us-doc-review and is now ready for technical review when you have a chance. Thank you!

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/barcode/setup/hardware.rst:143]: Small nitpick- add icon descriptor formatting around this too. I know this is an odd case since you are describing an app icon that is made up of multiple icons and colors.
```suggestion
:guilabel:`⚙️ (gear)` icon over a blue :guilabel:`(barcode)` icon.
```

**Review by samueljlieber — APPROVED** [review]
Hi @brse-odoo! Nice job on the improvements in this PR, everything looks good to me except for one very small nitpick 😄 please see below. I am approving now, but please address this one suggestion before passing to ZST for merge. Thank you!

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/barcode/setup/hardware.rst:79]: By @samueljlieber's logic this should be wrapped in parenthesis, as well, since, theoretically, an emoji would normally accompany the label (if there were one available for us to use).

```suggestion
represented by a :guilabel:`(light blue barcode)` icon).
```

**Review by StraubCreative — APPROVED** [review]
Hi @brse-odoo 
Thanks for the quick additions here. 
Approving with a comment from Sam, as well as one from me.
cc: @dara-odoo 

Merge when ready 👍 
..
@robodoo delegate=brse-odoo

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/barcode/setup/hardware.rst:143]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/barcode/setup/hardware.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/barcode/setup/hardware.rst:79]: Suggestion implemented ✅

**Comment by brse-odoo** [comment]
Thank you @samueljlieber and @StraubCreative for the quick tech changes, those two changes have been added and pushed up. Merging now 👍

**Comment by brse-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
